### PR TITLE
[CHI-2023] Set up sponsor page

### PR DESCRIPTION
**Pull request by mattstratton**
Signed-off-by: Matty Stratton <matty.stratton@aiven.io>

### CODE REVIEW

**Comment by mattstratton** [comment]
ping @albert-cheung for feedback

File: content/events/2023-chicago/sponsor.md [hunk 1]
```diff
@@ -4,63 +4,239 @@ Type = "event"
...
+      <th scope="col">PLATINUM<br>(Limit 2)</th>
+      <th scope="col">GOLD</th>
+      <th scope="col">BRONZE</th>
+      <th scope="col">COMMUNITY</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Price</td>
+      <td>$15,000</td>
```
Comment (kevinreedy) [inline content/events/2023-chicago/sponsor.md:33]: Can we show the discounted price here for now as well? Maybe something like:
```
      <td><del>$15,000</del>&nbsp$14,000</td>
      <td><del>$9,000</del>&nbsp$8,000</td>
      <td><del>$2,000</del>&nbsp$1,000</td>
```
Which looks like:
Screen Shot 2023-01-05 at 11 35 58 AM

Comment (mattstratton) [reply content/events/2023-chicago/sponsor.md:33]: oooh nice

Comment (mattstratton) [reply content/events/2023-chicago/sponsor.md]: see [84c79b0](https://github.com/devopsdays/devopsdays-web/pull/12118/commits/84c79b026cac1350396f49bc56f3eaa1abb0f6a5)

**Comment by kevinreedy** [comment]
Looks good to me. Paypal links look updated as well!

**Pull request merged**